### ISSUE TITLE
Disconnet after eats someone.

### ISSUE THREAD

**Issue by hunterpl** [huytd/agar.io-clone#139]
Hello Dear Developers

I test on my server dedication, your server files and when the player gamer eat someone, the eaten player has been disconnecting.

Best Regards
Mateusz

**Comment by hunterpl** [comment]
Add some crash log:


A user connected. Assigning UserID...
Player hfQfY_GDy1j30IHNAAAJ connected!
Total player: 2
User #hfQfY_GDy1j30IHNAAAJ disconnected

TypeError: Cannot read property 'name' of null
    at Socket.<anonymous> (/var/www/ts3bahuserwery/agar/bin/server/server.js:172:44)
    at Socket.emit (events.js:95:17)
    at Socket.emit (/var/www/ts3bahuserwery/agar/node_modules/socket.io/lib/socket.js:129:10)
    at Socket.onclose (/var/www/ts3bahuserwery/agar/node_modules/socket.io/lib/socket.js:418:8)
    at Client.onclose (/var/www/ts3bahuserwery/agar/node_modules/socket.io/lib/client.js:230:12)
    at Socket.emit (events.js:117:20)
    at Socket.onClose (/var/www/ts3bahuserwery/agar/node_modules/socket.io/node_modules/engine.io/lib/socket.js:248:10)
    at WebSocket.g (events.js:180:16)
    at WebSocket.emit (events.js:92:17)
    at WebSocket.Transport.onClose (/var/www/ts3bahuserwery/agar/node_modules/socket.io/node_modules/engine.io/lib/transport.js:113:8)
[00:33:27] [nodemon] app crashed - waiting for file changes before starting...

**Comment by themuddfamily2** [comment]
Duplicate of #151
